### PR TITLE
Update all API descriptions; more legalese

### PR DESCRIPTION
**Pull request by gwicke**
- Actually update the descriptions for all project specs.
- Add some more legalese by legal's request. While licensing is also
touched on in the ToUs, they feel that it won't hurt to make it really
explicit.

### CODE REVIEW

File: projects/wikimedia.org.yaml [hunk 1]
```diff
@@ -7,17 +7,27 @@ paths:
             version: 1.0.0-beta
```
Comment (Pchelolo) [inline projects/wikimedia.org.yaml]: Forgot `beta` here

File: projects/wmf_wiktionary.yaml [hunk 2]
```diff
@@ -8,17 +8,27 @@ paths:
             version: 1.0.0-beta
```
Comment (Pchelolo) [inline projects/wmf_wiktionary.yaml]: Forgot `beta` here

**Comment by Pchelolo** [comment]
We have the `common.yaml` spec that contains the reusable definitions. Can we maybe put this text there and merge it into the resulting spec?

**Comment by gwicke** [comment]
> We have the common.yaml spec that contains the reusable definitions. Can we maybe put this text there and merge it into the resulting spec?

Do you mean content.yaml? If so, that's at a lower (route) level. The description always comes from the top-level spec, which necessarily needs to differ between our projects.

**Comment by Pchelolo** [comment]
> Do you mean content.yaml? If so, that's at a lower (route) level. The description always comes from the top-level spec, which necessarily needs to differ between our projects.

Right, right, bas suggestion... 

LGTM +1

**Pull request merged**